### DISCUSSION TITLE
Hosting issue

### DISCUSSION THREAD

**Discussion by TEAQUAN** [vercel/community#6375] [Help]
### Summary

I hosted my website on vercel and used a payed domain name for the website but for some reason now when i search up the site its not working whats wrong and i am seeing some free team thing?

### Steps to Reproduce

https://peculiarconsulting.com/
this is the link

**Comment by amyegan** [comment]
Hey @TEAQUAN. It looks like [your domain is set up correctly](https://www.whatsmydns.net/#A/peculiarconsulting.com). And I see the site in search results from [Google](https://www.google.com/search?q=peculiar+consulting) and [DuckDuckGo](https://duckduckgo.com/?q=peculiar+consulting). I even checked [Bing](https://www.bing.com/search?q=peculiar+consulting) and your site was one of the top results. 😁 

Are you asking about something other than search engine results?

**Reply by TEAQUAN** [reply]
Anytime i tap on the link to load it up
It doesn't come up
Saying something along the lines of cannot connect to server
